packages/edit-site/src/components/global-styles/screen-revisions/index.js

```diff
@@ -95,7 +95,7 @@ function ScreenRevisions() {
 			<ScreenHeader
 				title={ __( 'Revisions' ) }
 				description={ __(
-					'Revisions are added to the timeline when style changes are saved.'
+					'Restore styles from an earlier version of changes that was automatically added to this timeline when new styles were saved.'
```

Comment:
Thanks for the PR!

The three clauses make it a bit of a mouthful in my opinion.

If we're seeking both clarity and guidance as mentioned in the [issue](https://github.com/WordPress/gutenberg/issues/55648), maybe we could:

1. describe what the user is looking at (the clarity)
2. provide instruction on how to restore styles (the guidance).

Not being prescriptive, these are just ideas. Language is fuzzy 😄 

E.g., 

This is pretty verbose:

> Click on previously saved styles to preview them. To restore a selected version to the editor, hit "Apply". When you're ready, save the template to commit your changes.

A bit more condensed:

> A timeline of saved changes to your site's styles. Click "Apply" to restore a selected version in the editor.

What do you reckon?

As a heads up, the revisions panel will be getting a UX makeover soon: https://github.com/WordPress/gutenberg/issues/55776

Reply:
I think the verbose option is the best. It explains better to the user what this panel do.
What do you think @pablohoneyhoney?

Reply:
I replace the text by the one you suggested 💪

Reply:
@sacerro wording looks good on the longer option, but the period needs to go inside the quotation mark like so:

>Click on previously saved styles to preview them. To restore a selected version to the editor, hit "Apply." When you're ready, save the template to commit your changes.

Reply:
Thanks @provenself 

I've updated the copy to reflect the latest changes. 

<img width="256" alt="Screenshot 2023-11-09 at 10 59 11 am" src="https://github.com/WordPress/gutenberg/assets/6458278/b1c9379f-5baf-4fe4-8575-84a8da3c83b4">


I'll merge this PR.

Thanks also to @sacerro !